composer.json

```diff
@@ -18,7 +18,7 @@
     "prefer-stable": true,
     "require": {
         "php": ">= 8.0",
-        "psr/simple-cache": "^2"
+        "psr/simple-cache": "^3"
```

Comment:
You should probably support both 2 and 3 like the approach I took in #5. A simple-cache v3 implementation is compatible with v2 because the types are more strict and that's the only change between releases. This makes the library more flexibility for developers to adopt and use the branch they need.

Reply:
Good point, I'll update this PR to support both version :+1: